sys/linux/mptcp.txt

```diff
@@ -46,21 +46,27 @@ mptcp_capable_option {
 mptcp_add_addr_option {
 	type	const[TCPOPT_MPTCP, int8]
 	length	len[parent, int8]
-	subtype	const[OPTION_ADD_ADDR, int8:4]
+# TODO: this was OPTION_ADD_ADDR, but it does not fit into 4 bits.
```

Comment:
@8litz There is something wrong with some of the consts here, or struct layout. These consts don't fit into 4 bits, also see below in this file. I've changed them to 0 for now as I wasn't able to figure out what's the right fix. This does not make things worse because they are truncated now anyway.
I will submit this soon. Please submit a proper fix afterwards. Thanks.

Reply:
@cpaasch fyi